jobs/service-fabrik-broker/templates/config/restartFabrikStatusPoller.js.erb

```diff
@@ -0,0 +1,2 @@
+const FabrikStatusPoller = require('/var/vcap/packages/service-fabrik-broker/lib/fabrik/FabrikStatusPoller.js')
```

Comment:
Having a js code inside templates is weird! Can we not make it part of the source code?

Reply:
Yes, would do.

Reply:
This should go away :-) FYI, Also this will not work since you are invoking this in another process. So another poller would be starting.